scala-package/README.md

```diff
@@ -124,11 +124,11 @@ Also, add the dependency which corresponds to your platform to the ```dependenci
 <dependency>
   <groupId>org.apache.mxnet</groupId>
   <artifactId>mxnet-full_2.11-osx-x86_64-cpu</artifactId>
-  <version>[1.9.0-SNAPSHOT,)</version>
+  <version>[1.9.1-SNAPSHOT,)</version>
 </dependency>
 ```
 
-**Note:** ```<version>[1.9.0-SNAPSHOT,)</version>``` indicates that we will fetch packages with version 1.9.0 or higher. This will always ensure that the pom.xml is able to fetch the latest and greatest jar files from Maven Snapshot repository.
+**Note:** ```<version>[1.9.1-SNAPSHOT,)</version>``` indicates that we will fetch packages with version 1.9.0 or higher. This will always ensure that the pom.xml is able to fetch the latest and greatest jar files from Maven Snapshot repository.
```

Comment:
```suggestion
**Note:** ```<version>[1.9.1-SNAPSHOT,)</version>``` indicates that we will fetch packages with version 1.9.1 or higher. This will always ensure that the pom.xml is able to fetch the latest and greatest jar files from Maven Snapshot repository.
```

Reply:
Good catch, thanks!